piecedirectory/piecedirectory.go

```diff
@@ -325,25 +327,16 @@ func (ps *PieceDirectory) BuildIndexForPiece(ctx context.Context, pieceCid cid.C
 	return nil
 }
 
-func (ps *PieceDirectory) DeleteDealForPiece(ctx context.Context, pieceCid cid.Cid, dealUuid uuid.UUID) error {
-	// ctx, span := tracing.Tracer.Start(ctx, "pm.delete_deal_for_piece")
-	// defer span.End()
-
-	// Delete deal from list of deals for this piece
-	//wasLast, err := ps.dealStore.Delete(pieceCid, dealUuid)
-	//if err != nil {
-	//return fmt.Errorf("deleting deal %s from store: %w", dealUuid, err)
-	//}
-
-	//if !wasLast {
-	//return nil
-	//}
-
-	//// Remove piece indexes
-	//if err := ps.deleteIndexForPiece(pieceCid); err != nil {
-	//return fmt.Errorf("deleting index for piece %s: %w", pieceCid, err)
-	//}
+func (ps *PieceDirectory) DeleteDealForPiece(ctx context.Context, pieceCid cid.Cid, dealUuid string) error {
```

Comment:
Let's rename DeleteDealForPiece to RemoveDealForPiece to be consistent with the service